packages/protocol/contracts/governance/Validators.sol

```diff
@@ -905,10 +649,7 @@ contract Validators is IValidators, Ownable, ReentrancyGuard, Initializable, Usi
 
     // Empty validator groups are not electable.
     if (groups[group].members.numElements == 0) {
-      if (votes.contains(group)) {
-        votes.remove(group);
-      }
-      emit ValidatorGroupEmptied(group);
+      getElection().markGroupIneligible(group);
```

Comment:
i didn't see the reverse operation on `addMember`!

Reply:
Discussed offline, will have `addMember` mark the group eligible automatically in a follow-up